src/Dev/DevBuildController.php

```diff
@@ -39,4 +51,29 @@ public function build(HTTPRequest $request): HTTPResponse
             return $response;
         }
     }
+
+    public function canInit(): bool
+    {
+        return (
+            Director::isDev()
+            // We need to ensure that DevelopmentAdminTest can simulate permission failures when running
+            // "dev/tasks" from CLI.
+            || (Director::is_cli() && DevelopmentAdmin::config()->get('allow_all_cli'))
+            || Permission::check("ADMIN")
+            || Permission::check("ALL_DEV_ADMIN")
+            || Permission::check("CAN_DEV_BUILD")
+        );
+    }
+    
+    public function providePermissions(): array
+    {
+        return [
+            'CAN_DEV_BUILD' => [
+                'name' => _t(__CLASS__ . '.CAN_DEV_BUILD_DESCRIPTION', 'Can execute dev/build'),
```

Comment:
```suggestion
                'name' => _t(__CLASS__ . '.CAN_DEV_BUILD_DESCRIPTION', 'Can execute /dev/build'),
```
The global override says "Can view and run all /dev endpoints" so these should all label themselves as `/dev/*`